File: Python/SmallestNumber.py
# Challenge 10: Smallest Number

# Find the Smallest number in an array.

# Example:
# [4, 10, 2, 8, 15, 3]

# Output:
# 2

number = [4, 10, 2, 8, 15, 3]
min = number[0]

for i in range(1, len(number)):
  if number[i] < min:
    min = number[i]

print(min)

File: JavaScript/SmallestNumber.js
/*
Challenge 10: Smallest Number

Find the Smallest number in an array.

Example:
[4, 10, 2, 8, 15, 3]

Output:
2
*/

let number = [4, 10, 8, 15, 3]
let max = number[0]

for (let i = 1; i < number.length; i++) {
  if (number[i] < max) {
    max = number[i]
  }
}

console.log(max)